backend/ibutsu_server/controllers/project_controller.py

```diff
@@ -99,12 +99,24 @@ def update_project(id_, project=None, token_info=None, user=None):
     if not is_uuid(id_):
         id_ = convert_objectid_to_uuid(id_)
     project = Project.query.get(id_)
-    user = User.query.get(user)
-    if project and project.owner.id != user.id:
-        return "Forbidden", 403
+
     if not project:
         return "Project not found", 404
-    project.update(connexion.request.get_json())
+
+    user = User.query.get(user)
+    if not user.is_superadmin:
```

Comment:
You could collapse this into a single `if`

```suggestion
    if not user.is_superadmin and project and project.owner.id != user.id:
```